docs/logs/getting-started-aspnetcore/Program.cs

```diff
@@ -0,0 +1,44 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Logs;
+using OpenTelemetry.Resources;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Logging.AddOpenTelemetry(logging =>
+{
+    logging.IncludeScopes = true;
+    logging.ParseStateValues = true;
+
+    var resourceBuilder = ResourceBuilder
+        .CreateDefault()
+        .AddService(builder.Environment.ApplicationName);
+
+    logging.SetResourceBuilder(resourceBuilder)
+        .AddConsoleExporter();
+});
+
+var app = builder.Build();
+
+app.MapGet("/", (ILogger<Program> logger) =>
```

Comment:
I don't know what's the actual recommendation from .NET team, do we want to use the DI logger or something different (e.g. https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging#log-in-programcs)?

Reply:
@davidfowl @noahfalk @tarekgh FYI

Reply:
that example from asp.net core docs looks confusing as it shows the approach of using `LoggerFactory` directly, but no other Asp.Net Core examples shows that approach!

Reply:
https://github.com/dotnet/AspNetCore.Docs/issues/30239 reported issue to asp.net core doc.